app/_layouts/extension.html

```diff
@@ -105,18 +132,51 @@
 {% capture plugin_config_for_service %}
 ### Enabling the plugin on a Service
 
-{% if page.params.yaml_examples == false and page.params.k8s_examples == false %}
+{% if page.params.yaml_examples == false and page.params.k8s_examples == false and page.params.manager_examples == false and page.params.konnect_examples == false %}
 
-{{ plugin_config_for_service_admin_api }}
+  {{ plugin_config_for_service_admin_api }}
 
 {% else %}
 
 {% navtabs %}
 {% navtab Admin API %}
 
-{{ plugin_config_for_service_admin_api }}
+  {{ plugin_config_for_service_admin_api }}
+
+  `<service>` is the `id` or `name` of the Service that this plugin
+    configuration will target.
 
 {% endnavtab %}
+{% unless page.params.manager_examples == false %}
+{% navtab Kong Manager %}
+Configure this plugin on a Service:
+
+1. In Kong Manager, select the workspace.
```

Comment:
Is there any way to  shorten and combine the nav steps?

Reply:
I did for Services and Consumers for Konnect, but for the others I cannot remove any of the steps as they are necessary.